index.js

```diff
@@ -439,6 +439,42 @@ Notifications.getDeliveredNotifications = function() {
   return this.callNative('getDeliveredNotifications', arguments);
 }
 
+Notifications.getScheduledLocalNotifications = function(callback) {
+	const mapNotifications = (notifications) => {
+		let mappedNotifications = [];
+		if(notifications?.length > 0) {
+			if(Platform.OS === 'ios'){
+				mappedNotifications = notifications.map(notif => {
+					return ({
+						soundName: notif.soundName,
+						repeatInterval: notif.repeatInterval,
+						id: notif.userInfo?.id,
```

Comment:
I'm a bit confused about this line,
I totally agree iOS should have the notification ID, but by default, `userInfo` is set to `{}`.
Not sure if we need to set a default value on `userInfo` (`notificationId` ?) or let this field empty when the developer doesn't set it in `userInfo`...

Reply:
That's why it's optional chained here. It would result in undefined if it is not set. Currently we're using `userInfo.id` for cancelling notifications by id, so it makes sense to me.

Reply:
But setting the id as default when scheduling would be an awesome feature IMHO.

Reply:
Will look into it, thanks for the PR !

Reply:
Good stuff! 🙏